app/utils/universalRaven.js

```diff
@@ -0,0 +1,31 @@
+//@flow
+
+export interface UniversalRaven {
+  captureException(ex: Error, options?: RavenOptions): *;
+  setDataCallback(data: any, orig?: any): *;
+  captureException(ex: Error, options?: RavenOptions): *;
+}
+
+export const createNewRavenInstance = (Raven: any): UniversalRaven =>
+  Raven.context(() => new UniversalRavenNode(Raven));
```

Comment:
@ekmartin 

This makes it a lot cleaner. Now you can call this to get a "custom" raven instance that can be passed around to all components dealing with the current request. Looks like it works well, but it should be tested some more I guess.